workos/organizations.py

```diff
@@ -47,114 +50,30 @@ def list_organizations(
         Returns:
             dict: Organizations response from WorkOS.
         """
-        warn(
-            "The 'list_organizations' method is deprecated. Please use 'list_organizations_v2' instead.",
-            DeprecationWarning,
-        )
-        if limit is None:
-            limit = RESPONSE_LIMIT
-            default_limit = True
 
         params = {
             "domains": domains,
             "limit": limit,
             "before": before,
             "after": after,
-            "order": order or "desc",
+            "order": order,
```

Comment:
With `order` previously being passed as `None`, this doesn't change the default ordering at all, right?

Reply:
Previously if `order` was not passed in, the value would fall back to `"desc"`.  Instead that default moves to the function signature: `order: PaginationOrder = "desc"`